Skip TestBuildNotVerboseFailure if no network

coolljt0725 opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/docker/docker/blob/master/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

For additional information on our contributing process, read our contributing
guide https://docs.docker.com/opensource/code/

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Please provide the following information:
-->

**\- What I did**

TestBuildNotVerboseFailure use a non-exist image `busybox1`,
it requires network connection to access to Dockerhub, skip
this test if there is no network.

Signed-off-by: Lei Jitang leijitang@huawei.com

thaJeztah commented:
Should we split the `{"quiet_build_not_exists_image", "FROM busybox11"},` to a separate test? Think that's the only one that's requiring network?

coolljt0725 commented:
@thaJeztah  I agree, will do

coolljt0725 commented:
@thaJeztah  Updated

thaJeztah commented:
LGTM, thanks!

vdemeester commented:
LGTM ♞

runcom commented on integration-cli/docker_cli_build_test.go:
```diff
@@ -4879,6 +4879,23 @@ func (s *DockerSuite) TestBuildNotVerboseSuccess(c *check.C) {
+	// and stderr output in verbose mode
+	testRequires(c, Network)
+	TestName := "quiet_build_not_exists_image"
+	BuildCmds := "FROM busybox11"
```
this uppercase ouch :smile:

thaJeztah replied:
Ah, yes, looks like it's copied from `TestBuildNotVerboseFailure` below, which was like that

coolljt0725 replied:
yes, it's copied from `TestBuildNotVerboseFailure` just to keep the var name consistent :smile:  
ok, I'll update

coolljt0725 commented:
@runcom  @thaJeztah  updated

coolljt0725 committed:
Skip TestBuildNotVerboseFailure if no network

TestBuildNotVerboseFailure use a non-exist image busybox1,
it requires network connection to access to Dockerhub, skip
this test if there is no network.

Signed-off-by: Lei Jitang <leijitang@huawei.com>

thaJeztah commented:
LGTM, and all 💚 merging
